### DISCUSSION TITLE
aws elasticache disables cluster mode read-write separation, whether it can be balanced for read endpoints？

### DISCUSSION THREAD

**Discussion by thangzhongyan** [redis/lettuce#2386] [Q&A]
#### Is your feature request related to a problem? Please describe

A clear and concise description of what the problem is. Ex. I have an issue when [...]

#### Describe the solution you'd like

A clear and concise description of what you want to happen. Add any considered drawbacks.

#### Teachability, Documentation, Adoption, Migration Strategy

If you can, explain how users will be able to use this and possibly write out a version the docs.
Maybe a screenshot or design?

**Comment by mp911de** [comment]
There's no built-in feature. You can provide a read-only connection pool along with balancing on top of Lettuce by implementing your own code.
